webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -817,8 +818,8 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
               />
             </ComparisonView>
             {showPagination && (
-              <Columns>
-                <Column align="right">
+              <Row>
+                <Column align="right" width="hug">
```

Comment:
fix: the pagination buttons are still left-aligned instead of right-aligned
![Screenshot 2023-11-14 at 16-31-21 Uncategorized Experiments - Determined](https://github.com/determined-ai/determined/assets/15078396/2a39b092-e9fb-490c-aa94-03abd35bc0a8)